### PR TITLE
Fix xml writing from Tile metadata

### PR DESCRIPTION
**Pull request by cbrrrry**
Some new math changes which reflect bug fixes in iSPIM acquisition. 
A lot of versioning thrash too. 
Is now working well for new iSPIM data. 
Going to be pulling out the iSPIM chron job features for a separate PR shortly.

### CODE REVIEW

File: src/aind_data_transfer/jobs/zarr_upload_job.py [hunk 1]
```diff
@@ -203,22 +205,98 @@ def _upload_zarr(self) -> None:
...
+            log_dict = {'imaging_log_file': None} #set this to none to read schema_log
+
+
+        if acq_file.exists():
+            acq_json = read_dispim_aquisition(acq_file)
+            
+
+        #if any of the values of log_dict are None, then get it from schema_log
+        elif any(v is None for v in log_dict.values()):
+            print('Some values in imaging_log.log are None. Reading schema_log.log')
```
Comment (carshadi) [inline src/aind_data_transfer/jobs/zarr_upload_job.py:271]: can we replace the print statements with the instance logger?

File: src/aind_data_transfer/jobs/zarr_upload_job.py [hunk 2]
```diff
@@ -203,22 +205,98 @@ def _upload_zarr(self) -> None:
...
+
         # convert to acq json
-        acq_json = log_to_acq_json(log_dict)
         acq_json_path = self._data_src_dir.joinpath('acquisition.json')
 
+
         try:
             write_acq_json(acq_json, acq_json_path)
-            self._instance_logger.info('Finished writing acq json')
+            print('Finished writing acq json')
```
Comment (carshadi) [inline src/aind_data_transfer/jobs/zarr_upload_job.py:297]: same here

**Review by carshadi — CHANGES_REQUESTED** [review]

**Comment by carshadi** [comment]
@cbrrrry tests are failing, but otherwise looks good to me!

File: src/aind_data_transfer/jobs/zarr_upload_job.py (hunk 1 above)
Comment (cbrrrry) [reply src/aind_data_transfer/jobs/zarr_upload_job.py:271]: Yes, good point. Will do!

**Comment by cbrrrry** [comment]
> @cbrrrry tests are failing, but otherwise looks good to me!

I tested the zarr_upload_test.py file, and it's passing. It's the ephys tests that are failing, which is interesting. Recommendations?

**Review by carshadi — APPROVED** [review]

**Pull request merged**